### ISSUE TITLE
is this abandoned?

### ISSUE THREAD

**Issue by fserb** [hayd/deno-udd#116]
It would be a shame. It's such a useful tool.

**Comment by e-kwsm** [comment]
deno now supports [`outdated`](https://docs.deno.com/runtime/reference/cli/outdated/) command to find and update outdated dependencies, but it works only against deno.json and package.json.

**Comment by Shougo** [comment]
You can use molt instead.

https://github.com/hasundue/molt

**Comment by fserb** [comment]
yeah, `deno outdated` only works for `deno.json` deps, not in file deps.

But yeah, molt seems awesome, thanks for the pointer! :)
